src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransactionContent.tsx

```diff
@@ -93,26 +93,40 @@ const GroupedTransactionContent: FC<GroupedTransactionContentProps> = ({
             toggleCancelConfirmation={toggleCancelConfirmation}
           />
         ) : (
-          <TransactionStatus status={status} />
+          <TransactionStatus status={status} hasError={!!error} />
```

Comment:
I have some reservations about passing in a separate error here, but I think it's necessary for now. I will handle this in my refactor.